tests/e2e/local-chrome.conf.js

```diff
@@ -28,7 +28,7 @@ const nightwatch_config = {
         "disable-web-security",
         "ignore-certificate-errors",
         "no-sandbox",
-        "headless",
+        // "headless", // headless is not possible as it doesn't allow to fix the internationalization
```

Comment:
O.o Could you please explain this bit?

Reply:
a number like `0.4` is displayed in Germany `0,4` so when you compare both in the UI it would fail.

Reply:
oh so interesting!